contents/why_julia.md

```diff
@@ -53,7 +53,7 @@ We reserved @sec:julia_wild for displaying successful Julia use cases in both ac
 **In Julia you can do your custom different stuff without loss of performance**.
 
 3. Had to debug code and somehow you see yourself reading a Fortran or C/C++ source code and having no idea what you are trying to accomplish?
-**In Julia you only read Julia code[^readable], no need to learn another language to make your original language fast**.
```

Comment:
This footnote is referred to twice (also a bit lower on the page).

Reply:
Yes this is not a bug or error. It is intentional